src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int ONE_ARRAY_INDEX = 1;
```

Comment:
Unclear constant name. What does it stand for? The index of the cat picture, working hours amount, operation name, or something else? Provide a concrete name that reflects the purpose of the value with such an index following your task requirements